src/App.vue

```diff
@@ -1,13 +1,40 @@
 <template>
 	<v-app>
+		<v-navigation-drawer
```

Comment:
Navigation Drawers themselves are used pretty much once or twice in the entire app, hence abstraction is not a suitable task for Navigation Drawers in general.

Reply:
Yes, provided that this will be released as a free and open source software for the world, it is important to keep the components clear and abstracted out :)

Reply:
I guess that's fair. In the event we do use it multiple times later, we will probably need to be careful about abstracting it out!